lib/bepaid/gateway_behaviour.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Bepaid.GatewayBehaviour do
+  @moduledoc """
+  Interface for Gateway. Could be used for defining your own module for testing
+  """
+
+  alias Bepaid.Payment
+
+  @type http_response :: {:ok, map} | {:error, any}
+
+  @doc """
+  """
```

Comment:
@MikeAndrianov is this a good place to provide docs? If so, I will input description here. Please let me know.

Reply:
As I can see in ecto repo, they cover behaviours [with docs](https://github.com/elixir-ecto/ecto/blob/master/lib/ecto/type.ex), but maybe we could get rid of this for now.